### DISCUSSION TITLE
[Bug]: Publish options goes away once adding Zoom url

### DISCUSSION THREAD

**Discussion by eflashlogistics** [code4recovery/12-step-meeting-list#737] [Q&A] [answered]
### Contact Details

lucas.3@aol.com

### Website With Issue

http://lucasg11.sg-host.com/

### What happened?

Publish button disappears once I enter the zoom URL. tested multiple url's

https://www.loom.com/share/6dfcd156ded04fc5b173d95e4a78afcb

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### What browsers are you seeing the problem on?

Chrome

**Comment by tech2serve** [comment, marked as answer]
Please open Discussions and leave the Issues for the developers.

I appreciate the screen cast, unfortunately, it didn't show the entire process.

Please make sure when you're entering an online meeting (i.e., adding the Zoom link) that you also add a city and state (and only a city and state, or county and state) so the address is will come back from Google as approximate. This is required in order to publish online meetings.

If you need further information, there are a number of Discussion threads covering how to add online meetings. Please use the search feature here, and you should them.

Please let us know if this resolves your issue, or you need further help.

**Comment by eflashlogistics** [comment]
Ahh I see. Honestly my dev team uses github and I have it but somewhat new 

I see the discussion area now. 

I think the problem is that was the whole thing

I will add those other parameters and I am sure it will work. Thanks.
